src/components/ImportFile/ImportFile.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { ImportFile, ImportFileContent } from './index';
+
+const files = [];
+
+test('ImportFile is working properly', () => {
+  const component = shallow(
+    <ImportFile acceptedTypes="image/png" maxAmount={4} maxSize={4000} />
+  );
+  component.instance().onDrop(files, files);
+  component.instance().reset();
+  component.instance().onDropAccepted();
+  component.instance().onDropRejected();
+  component.instance().onDragEnter();
+
+  expect(component.render()).toMatchSnapshot();
```

Comment:
no need for render in snapshots.  This project is using enzyme-to-json's snapshot serializer.